app/controllers/api/api_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::ApiController < ApplicationController
+  def random_greeting
+    random_message = Message.order('RANDOM()').first
+    render json: { greeting: random_message&.message || 'No greetings available' }
+  end
+end
```

Comment:
- [ ] Amazing job working on the backend side but I think you didn't complete the setup with the frontend side since you should link your API with the frontend side through greeting component action, reducer, and store so It would be great if you created them and linked them like a react app inside the project as usual, so please add them as mentioned in the requirements.  ✨🚀🔥

- Your messages are displayed but not by the component and the react app but once you link the API with the comment through your store js the messages will be displayed from there 🎉🎉🎉🎉🎉  

![image](https://github.com/sergio-usma/hello_react_rails/assets/58553711/96232b3a-4146-4cbf-b058-a2fa7b673c17)